recipes/leftbar/general_row/general_row.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div
+    :class="leftbarGeneralRowClasses"
+    data-qa="dt-leftbar-row"
+  >
+    <a
+      class="dt-leftbar-row__primary"
+      data-qa="dt-leftbar-row-link"
+      v-on="$listeners"
+    >
```

Comment:
I happened to default to `<a>`, though it supports `<button>` too — and implicitly `<el role="button">`. What's your call on how it should be demo'ed in these recipes?

Meanwhile, omitting `href` removes focusability. if you do decide to retain as `<a>`, include a dummy `href="#link-to-nowhere"`.

And looking forward to the other recipes in [DT-934](https://dialpad.atlassian.net/browse/DT-934), you'll probably want to put a note in there to emulate the keyboard focus script in that codepen.

Reply:
Aside: are [these really three different recipes](https://dialpad.atlassian.net/browse/DT-934) though?

Reply:
I just changed this to a button for now since it seems there isn't really a way to fire an event with no action from a link (it is supposed to directly link to something). I'd rather not put `#link-to-nowhere` in there as it changes url history.

Reply:
> Aside: are [these really three different recipes](https://dialpad.atlassian.net/browse/DT-934) though?

We decided to do this pretty much solely for the purpose of simplicity, only exposing properties the consumer is likely to need. It's definitely possible for it to be a single component. As it is now I would likely use general row as a base for the two others.

Reply:
Cool, figured it'd be extended by others. I'm sure the other ones will get done right quick.